### ISSUE TITLE
applying formatting suppresses events

### ISSUE THREAD

**Issue by AsherBarak** [BobKnothe/autoNumeric#180]
applying formatting causes this not to execute:

```javascript
 $(selection).on('input propertychange paste', function (event) { ... }) 
```

Updating other values while the user types in a formatted textbox is a common scenario (think updating sums etc.).

**Comment by BobKnothe** [comment]
Asher,

I have updated 2.0 and improved the shim to throw the input event. 

Would appreciate any feedback and suggeztions.

Bob

**Comment by BobKnothe** [comment]
Fixed in version 2.0
